joern-cli/frontends/jssrc2cpg/src/test/scala/io/joern/jssrc2cpg/dataflow/DataflowTest.scala

```diff
@@ -586,4 +586,15 @@ class DataflowTest extends DataFlowCodeToCpgSuite {
     println(sink.reachableBy(src).l)
   }
 
+  "Flow from inside object notation to call argument" in {
+    val cpg: Cpg = code("""
+        |const a = { b : 47 } ;
+        |fn(a);
+        |""".stripMargin)
+
+    val sink = cpg.call.nameExact("fn")
+    val src  = cpg.literal("47")
+    sink.reachableBy(src).nonEmpty shouldBe true
```

Comment:
Please make this `sink.reachableBy(src).size shouldBe 1` so that we can detect if we ever get more than one flow here as well.